### PR TITLE
Add images to steps under "Permanent protection"

### PR DESCRIPTION
**Pull request by Priyankag16**

### CODE REVIEW

**Comment by Priyankag16** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/android-support-signin.md [hunk 1]
```diff
@@ -120,9 +120,23 @@ We are working with OEM to find a solution to enable this permission from the ap
 Users can follow these steps to enable the same permissions from the device settings: 
 
 1. Go to **Settings** on your device.
+   
 2. Search for and select **Battery Optimization**.
+   
+   ![Search and select "Battery Optimisation](images/searchBatteryOptimisation.png)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/android-support-signin.md]:
```suggestion
   ![Search for and select "Battery Optimisation"](images/searchBatteryOptimisation.png)
```

File: microsoft-365/security/defender-endpoint/android-support-signin.md [hunk 2]
```diff
@@ -120,9 +120,23 @@ We are working with OEM to find a solution to enable this permission from the ap
...
+
 3. In **Special app access**, select **Battery Optimization**.
+   
+   ![In Special app access, select "Battery Optimisation"](images/specialAppAccess.png)
+
 4. Change the Dropdown to show **All Apps**.
+
+   ![Change dropdown to show All Apps](images/showAllApps2.png)
+
+   ![Change dropdown to show All Apps](images/showAllApps1.png)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/android-support-signin.md]:
```suggestion
   ![Change dropdown to show "All Apps"](images/showAllApps2.png)

   ![Change dropdown to show "All Apps"](images/showAllApps1.png)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Priyankag16, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**